src/PharoThings-Hardware-Core.package/PotDevice.class/instance/configureDefaultPeripherals.st

```diff
@@ -0,0 +1,6 @@
+initialization
+configureDefaultPeripherals
+   "Subclasses can provide example configuration of pin variables.
+   So users will be able simply create device using #new (PotLCD new)
+   and see in the board inspector how to connect it in the real world (with real wires)"
+	self subclassResponsibility
```

Comment:
It should not be mandatory to implement such method. 
I would make it empty because for many devices it is redundant as we discussed

Reply:
Do you think that purpose of default config is to be only an example? Or is it something more?
If it's just an example I would name this method accordingly: #configureExamplePeripherals.
I feel that the meaning of #default includes some functional responsibility but example is not about it

Reply:
I think we can remove Default or Example and leave only #configurePeripherals, because it is a real configuration.